test/k8s/istio.go

```diff
@@ -1,430 +0,0 @@
-			if res.WasSuccessful() {
-				GinkgoPrint("Was able to connect from %q to %q, but expected no connection: %s", srcPod, dstURI, res.OutputPrettyPrint())
-				return false
-			}
-			return true
-		}
-
-		// formatLabelArgument formats the provided key-value pairs as labels for use in
-		// querying Kubernetes.
-		formatLabelArgument := func(firstKey, firstValue string, nextLabels ...string) string {
-			baseString := fmt.Sprintf("-l %s=%s", firstKey, firstValue)
-			if nextLabels == nil {
-				return baseString
-			} else if len(nextLabels)%2 != 0 {
-				Fail("must provide even number of arguments for label key-value pairings")
-			} else {
-				for i := 0; i < len(nextLabels); i += 2 {
-					baseString = fmt.Sprintf("%s,%s=%s", baseString, nextLabels[i], nextLabels[i+1])
-				}
-			}
-			return baseString
-		}
-
-		// formatAPI is a helper function which formats a URI to access.
-		formatAPI := func(service, port, resource string) string {
-			target := fmt.Sprintf(
-				"%s.%s.svc.cluster.local:%s",
-				service, helpers.DefaultNamespace, port)
-			if resource != "" {
-				return fmt.Sprintf("%s/%s", target, resource)
-			}
-			return target
-		}
-
-		// formatLocalAPI is a helper function which formats a URI to access.
-		formatLocalAPI := func(port, resource string) string {
-			target := fmt.Sprintf("http://127.0.0.1:%s", port)
-			if resource != "" {
-				return fmt.Sprintf("%s/%s", target, resource)
-			}
-			return target
-		}
-
-		outbound := "outbound"
-		inbound := "inbound"
-
-		// shouldHaveService checks that srcPod has service properly configured.
-		shouldHaveService := func(pod, service, port, direction string) bool {
-			var target string
-			if service != "" {
-				target = fmt.Sprintf("%s.%s.svc.cluster.local", service, helpers.DefaultNamespace)
-			}
-			By("Checking that Istio proxy config at %q has service %q on port %q for %q", pod, target, port, direction)
-			res := kubectl.Exec(fmt.Sprintf(`./cilium-istioctl proxy-config cluster %s | grep "%s.*%s.*%s"`, pod, target, port, direction))
-			if !res.WasSuccessful() {
-				GinkgoPrint("Service %q on port %s for %s not configured at %q", target, port, direction, pod)
-				return false
-			}
-			return true
-		}
-
-		SkipItIf(helpers.RunsOnAKS, "Tests bookinfo inter-service connectivity", func() {
-			var err error
-			version := "version"
-			v1 := "v1"
-
-			productPage := "productpage"
-			reviews := "reviews"
-			ratings := "ratings"
-			details := "details"
-			dnsChecks := []string{productPage, reviews, ratings, details}
-			app := "app"
-			health := "health"
-			ratingsPath := "ratings/0"
-			apiPort := "9080"
-			podNameFilter := "{.items[*].metadata.name}"
-
-			bookinfoV1YAML := helpers.ManifestGet(kubectl.BasePath(), "bookinfo-v1.yaml")
-			bookinfoV2YAML := helpers.ManifestGet(kubectl.BasePath(), "bookinfo-v2.yaml")
-			l7PolicyPath := helpers.ManifestGet(kubectl.BasePath(), "cnp-specs.yaml")
```

Comment:
I think we can also remove these manifest files from `test/k8s/manifests` as part of this PR.